app/views/layouts/_header.html.erb

```diff
@@ -34,10 +34,6 @@
     <%= render(PhaseBannerComponent.new(
       feedback_link: ProviderInterface::FEEDBACK_LINK,
     )) %>
-  <% elsif controller.controller_name.in? %w[sessions content provider_agreements] %>
```

Comment:
Should we only be hiding this for content, as I believe we don't want providers navigating away when viewing provider agreements, and also sessions includes signing in etc? Also I think removing this does not just show the primary navigation but also the application list and reports which we would like to hide

Reply:
Thanks. Have updated so this change is only for content pages and not sessions or provider agreements.

Re what should appear in the primary nav, I've messaged Adam to get his opinion. AFAIK the provider primary nav is only defined [here](https://github.com/DFE-Digital/apply-for-teacher-training/blob/54fcc614f6755c8920d42f6b188ff342bed85e93/app/helpers/navigation_items.rb#L50), which incorporates application list and reports.